.github/workflows/release.yaml

```diff
@@ -0,0 +1,89 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.*'
+
+jobs:
+  build:
+    name: Build Binaries
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        goos: [linux, windows, darwin]
+        goarch: [amd64, arm64]
+        wasm-tools-version: ["1.216.0"]
+        exclude:
+          - goos: windows
+            goarch: arm64
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+
+      - name: Set up wasm-tools
```

Comment:
Why does this workflow need `wasm-tools` at all?

Reply:
yeah you're right. It's not needed. I thought `go test ./...` requires wasm-tools but I was wrong.